launch/tier4_localization_launch/launch/util/util.launch.xml

```diff
@@ -4,6 +4,11 @@
   <arg name="input/pointcloud" description="input topic name"/>
   <arg name="output/pointcloud" default="downsample/pointcloud" description="final output topic name"/>
 
+  <let name="ndt_enabled" value="true" if="$(eval &quot;'$(var pose_estimator_mode)'=='lidar'&quot;)"/>
+  <let name="ndt_enabled" value="false" if="$(eval &quot;'$(var pose_estimator_mode)'=='gnss'&quot;)"/>
```

Comment:
`ndt_enabled` is defined but not used. 
Currently, `ndt_enabled` is set by [pose_initializer.param.yaml](https://github.com/autowarefoundation/autoware_launch/blob/34e25d982e708fc73d454f531237b156f5f2126e/autoware_launch/config/localization/pose_initializer.param.yaml)

Reply:
I had overlooked that change. Fixed.